core/lib/permissions.ts

```diff
@@ -18,3 +18,39 @@ export const getPubUsers = (permissions: PermissionPayload[]) => {
 	}
 	return users;
 };
+
+enum Privileges {
+	movePub,
+	createPub,
+	viewPub,
+	updatePubValues,
+	createRelatedPub,
+	createPubWithForm,
+	editPubWithForm,
+	createPubField,
+	archivePubField,
+	editPubField,
+	createPubType,
+	editPubType,
+	deletePubType,
+	createStage,
+	deleteStage,
```

Comment:
should we also have `editStage`, for changing the name? (and possibly other stage level configuration in the future?)

Reply:
good point. and i think that can encompass all of the actions/rules config too.